src/Server.php

```diff
@@ -31,20 +31,34 @@
      */
     private $parameters;
 
+    /**
+     * @var int Connection options
+     */
+    private $options;
+
+    /**
+     * @var int Retries number
+     */
+    private $retries;
+
     /**
      * Constructor.
      *
      * @param string $hostname   Internet domain name or bracketed IP address
      *                           of server
      * @param string $port       TCP port number
      * @param string $flags      Optional flags
+     * @param int $options       Connection options
+     * @param int $retries       Retries number
      * @param array  $parameters Connection parameters
```

Comment:
Parameters should be ordered accordingly to method signature.
Moreover you need to run `vendor/bin/php-cs-fixer fix -v`